### DISCUSSION TITLE
Still Required to have Apple Mac or IOS to use farcaster?

### DISCUSSION THREAD

**Discussion by fblauer** [farcasterxyz/protocol#45] [Q&A]
Is it still not possible to use farcaster unless you have Mac or IOS? I am using Web/Linux/Android (or even windows if I have to). No apple hardware here

**Comment by varunsrin** [comment]
There is an android beta for the Merkle Manufactory client, DM @varunsrin on Twitter for the link

**Comment by fblauer** [comment]
I am using it and it works,  but I really would prefer desktop. Thanks

**Reply by hellno** [reply]
I've been using this website on desktop https://warpcast.com
